modules/data_release/php/NDB_Menu_Filter_data_release.class.inc

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * This file handles the Data Release for LORIS
+ *
+ * PHP Version 5
+ *
+ * @category LORIS
+ * @package  Main
+ * @author   Justin Kat <justinkat@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris
+ */
+
+require_once 'NDB_Menu_Filter.class.inc';
+
+/**
+ * Data Release Class
+ *
+ * This class is for the Data Release
+ *
+ * @category LORIS
+ * @package  Main
+ * @author   Justin Kat <justinkat@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris
+*/
+
+class NDB_Menu_Filter_Data_Release extends NDB_Menu_Filter
+{
+    /**
+     * Setup variables
+     *
+     * @return None
+     */
+    function _setupVariables()
+    {
+        $user =& User::singleton();
+
+        // set the class variables
+        $this->columns      = array(
+                               'file_name',
+                               'version',
+                               'upload_date',
+                              );
+        $this->query        = " FROM data_release WHERE id IN"
+                              . " (SELECT data_release_id FROM"
+                              . " data_release_permissions WHERE"
+                              . " userid=(SELECT ID FROM users WHERE"
+                              . " UserID='{$user->getUsername()}'))";
```

Comment:
This is a really awkward query.. can it be replaced by a join? Also, string concatentation in an SQL = injection attack.